src/Component.php

```diff
@@ -210,18 +211,18 @@ protected function logExecutedSqls(): void
     /**
      * @throws \Keboola\Component\UserException
      */
-    protected function executeStep(string $step): void
+    protected function executeStep(string $step, DwhLocationEnum $dwhLocation): void
     {
         $this->getLogger()->info(sprintf('Executing command "%s"', $step));
         $dbtService = new DbtService($this->projectPath);
```

Comment:
Nebylo by lepší to poslat do konstruktoru jako property ať to tam nepropadává pak všema funkcema v argumentech?

Reply:
Souhlas, bude to vypadat lépe, upravil jsem